Misc/NEWS.d/next/Core_and_Builtins/2025-02-12-01-36-13.gh-issue-129858.M-f7Gb.rst

```diff
@@ -0,0 +1,13 @@
+`elif` statements that follow an `else` block now have a specific error message.

```

Comment:
This one should be done in the What's New page (in Doc/whatsnew/3.14.rst). You can leave out the example in the blurb and keep the message.

Reply:
Note: I will be offline for a few days so I'll review the changes this w-e!

Reply:
Enjoy your time offline! (That is, I hope you have enjoyed it by the time you read this.)